### PR TITLE
Update config + small UI changes

### PR DESCRIPTION
**Pull request by caseycesari**
## Overview

Updates the site config with the real regions. Also makes some other small UI changes. See commit messages for details.

Connects #7 
Connects #9 

## Demo

tnc1

## Testing Instructions

- Browse the sidebar and verify that all of the regions currently on `maps.coastalresilience.org` are in the sidebar.
- Verify that clicking a subregion launches the site.

### CODE REVIEW

File: src/config.json [hunk 1]
```diff
@@ -15,54 +15,121 @@
             "mapZoom": 3,
             "subregions": [
                 {
-                    "title": "United States",
-                    "image": "assets/images/locations/US-Mainland.svg",
-                    "mapUrl": "https://coastalresilience.org/"
+                    "title": "Global",
+                    "image": "assets/images/locations/globe.svg",
+                    "mapUrl": "https://maps.coastalresilience.org/global"
```
Comment (kellyi) [inline src/config.json]: Having trouble getting this URL to resolve on clicking the link.

File: src/config.json [hunk 2]
```diff
@@ -15,54 +15,121 @@
...
             "mapZoom": 3,
             "subregions": [
                 {
-                    "title": "United States",
-                    "image": "assets/images/locations/US-Mainland.svg",
-                    "mapUrl": "https://coastalresilience.org/"
+                    "title": "California",
+                    "description": "Helping California’s coastal communities plan for resilience in the face of sea level rise",
+                    "image": "assets/images/locations/california.svg",
+                    "mapUrl": "https://maps.coastalresilience.org/california"
```
Comment (kellyi) [inline src/config.json]: These US subregion URLs also don't resolve very quickly if at all.

**Comment by kellyi** [comment]
Checked this in IE11, FF, and Safari along with Chrome and it's working in each, with an element-positioning quirks in FF:

In FF the close button for the region header is positioned slightly off:

screen shot 2017-10-31 at 1 50 12 pm

File: src/config.json (hunk 2 above)
Comment (caseycesari) [reply src/config.json]: Ah good catch. Shouldn't be `https`.

**Comment by caseycesari** [comment]
Re: menu button in sidebar, I'll create an issue.

**Comment by caseycesari** [comment]
Pushed a fixup for the region links.

**Comment by kellyi** [comment]
Taking another look at this.

**Review by kellyi — APPROVED** [review]
+1. Links are now working, along with everything else.

**Comment by caseycesari** [comment]
Thanks for the review!

**Pull request merged**